region/maridia/outer.json

```diff
@@ -492,6 +774,28 @@
                       "requires": []
                     }
                   ]
+                },
+                {
+                  "name": "G-Mode Morph Breaking the Maridia Tube Gravity Jump",
+                  "notable": true,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [1,3],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    {"ammo": { "type": "PowerBomb", "count": 1 }},
```

Comment:
This cost is part of the breaking glass event.  But then I'm not sure how that interacts with `not: f_MaridiaTubeBroken`.
I'd guess you could follow these gravity jump strats to reach top door repeatedly, and get back down with maridiaTubeClip, and never really break the tube to get the flag. Because that would add underwater movement requirements for future trips.
Seems like the `yield` needs to be attached to these strats.

Reply:
If I'm understanding this right, that does seem to be a problem ... It's the same as the non g-mode version of the same strat, which Map Rando doesn't yet use because of `canRiskPermanentLossOfAccess`. I'll talk to Maddo about this as well. I know that currently we can't include a `yields` on a strat, but that would be nice in several places and possibly required here.

Reply:
Apparently there are problems with having a `yields` on a link. This new commit should fix these problems.